templates/vue/cypress/integration/tyde.spec.js

```diff
@@ -0,0 +1,307 @@
+      cy.getNodeByTitle("Question Set 1").then(node => {
+        cy.getNodeById(node.id).click()
+        cy.openLightbox(node.id)
+        cy.getByTestId("favourite-button").click()
+        cy.closeLightbox()
+        cy.openTydeMenu()
+        cy.getByTestId("tyde-favourites-button").click()
+        cy.contains(/Question Set 1/i).should("exist")
+        cy.getByTestId("tyde-unfavourite").click()
+        cy.contains(/Yes/i).click()
+        cy.contains(/You have not added any items to your favourites/i).should(
+          "exist"
+        )
+      })
+    })
+
+    it("Should show help info in the cockpit", () => {
+      cy.fixture("one-node.json").as("oneNode")
+      cy.setup("@oneNode")
+      cy.openTydeMenu()
+      cy.getByTestId("tyde-help-button").click()
+      cy.contains(/ask@tyde.ca/i).should("exist")
+    })
+
+    it("Should navigate to the summary view when clicking a completed module", () => {
+      cy.fixture("tyde/one-question-set-part-images.json").as("oneQuestionSet")
+      cy.setup("@oneQuestionSet")
+      cy.getNodeByTitle("Question Set 1").then(node => {
+        cy.getNodeById(node.id).click()
+        cy.openLightbox(node.id)
+        cy.getByTestId("favourite-button").click()
+        cy.closeLightbox()
+      })
+      cy.openTydeMenu()
+      cy.getByBackground(earnedCockpitIcon).click()
+      cy.getBySrc(earnedIcon).should("exist")
+      cy.contains(/Module 1/i).should("be.visible")
+      cy.contains(/Stage 1/i).should("be.visible")
+      cy.contains(/Question Set 1/i).should("be.visible")
+      cy.contains(/Activities/i).click()
+      cy.contains(/Favourites/i).click()
+      cy.contains(/Question Set 1/).should("be.visible")
+    })
+
+    it("Should show completed profile activities in the menu", () => {
+      cy.fixture("tyde/activities.json").as("activities")
+      cy.setup("@activities")
+      //cy.openModal("settings")
+      // Must force here due to the menu icon covering the settings icon
+      cy.getByTestId("settings-button").click({ force: true })
+      cy.getByTestId("settings-modal")
+        .contains(/profile/i)
+        .click()
+      cy.contains(/add activity/i).click()
+      cy.getByTestId("profile-activity-combo").click()
+      cy.contains("What's your name?").click()
+      cy.submitSettingsModal()
+      cy.openTydeMenu()
+      cy.getByTestId("tyde-menu-profile-button").click()
+      cy.contains("What's your name?").should("not.exist")
+      cy.closeTydeMenu()
+      cy.getNodeByTitle("Activity 1").then(node => {
+        cy.getNodeById(node.id).click()
+        cy.openLightbox(node.id)
+        cy.getByTestId("record").click()
+        cy.contains(/done/i).click()
+        cy.contains(/submit/i).click()
+        cy.contains("Your response has been recorded.").should("be.visible")
+        cy.closeLightbox()
+      })
+      cy.openTydeMenu()
+      cy.getByTestId("tyde-menu-profile-button").click()
+      cy.contains("What's your name?").should("exist")
```

Comment:
@wynnset This assertion fails right now. Can you confirm what is actually supposed to happen here? After that is figured out, this is ready for review.

Reply:
@rlnsy In the profile, the answer the user provided to the given question is supposed to show up. If you are answering using audio, then an audio element should exist with the recorded audio. Here you are checking for the question, but I think you need to be checking for the answer.